browser/css/mobilewizard.css

```diff
@@ -3,8 +3,8 @@ label.mobile-wizard {
 	float: left;
 }
 
-.menuwizard .menu-entry-icon{
```

Comment:
this change causes regression in character wizard
![image](https://github.com/CollaboraOnline/online/assets/26241069/69ffaebf-72cf-4b11-8a65-a46d506cdf7a)